Singularity/Singularity/Screen/ScreenClasses/AchievementsScreen.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace Singularity.Screen.ScreenClasses
+{
+    /// <inheritdoc cref="IScreen"/>
+    /// <summary>
+    /// Used to show achievements that the player has earned.
+    /// It shows achievements already earned in a list format, not
+    /// unsimilar to how steam shows achievements. It has no buttons
+    /// other than back and simply shows textures of the achievements
+    /// and their description. If an achievement has not been earned yet,
+    /// the achievement texture is blacked out.
+    /// </summary>
+    class AchievementsScreen : IScreen
+    {
+        /// <summary>
+        /// Updates the contents of the screen.
+        /// </summary>
+        /// <param name="gametime">Current gametime. Used for actions
+        /// that take place over time </param>
+        public void Update(GameTime gametime)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Draws the content of this screen.
+        /// </summary>
+        /// <param name="spriteBatch">spriteBatch that this object should draw to.</param>
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Loads any content specific to this screen.
+        /// </summary>
+        /// <param name="content">Content Manager that should handle the content loading</param>
+        public void LoadContent(ContentManager content)
+        {
+            // TODO
+        }
+
+        /// <summary>
+        /// Determines whether or not the screen below this on the stack should update.
+        /// </summary>
+        /// <returns>Bool. If true, then the screen below this will be updated.</returns>
+        public bool UpdateLower()
+        {
+            throw new NotImplementedException();
```

Comment:
maybe we should get an abstract default Screen for those ....

Reply:
No because each screen has a different update lower and draw lower value

Reply:
ignore that, I was just confused